### ISSUE TITLE
Button group style inconsistent?

### ISSUE THREAD

**Issue by jfrankl** [PublicMapping/districtbuilder#438]
## Description
I just noticed that the selection and geounit button groups have slightly different amounts of spacing between them. @alexelash do you know if this is intentional?

![Screen Shot 2020-09-21 at 4 39 47 PM](https://user-images.githubusercontent.com/1809908/93819137-59487180-fc29-11ea-90db-23cd580ac499.png)

**Comment by alexelash** [comment]
@jfrankl It's a mix of intentional and not intentional—we had briefly talked about this way back. I had hoped to make them like the demo, but was having trouble styling the first and last button like end caps so I kept spacing between them. At the time you had mentioned you had a fix for it in your PR. Maybe we had crossed wires, but either way I forgot to follow up on it so I'm glad you're bringing it up. 

Long and short is: it'd be better if they were the same.
